src/backend/distributed/utils/metadata_cache.c

```diff
@@ -98,6 +101,10 @@ static Oid distTransactionRelationId = InvalidOid;
 static Oid distTransactionGroupIndexId = InvalidOid;
 static Oid extraDataContainerFuncId = InvalidOid;
 
+/* Citus installed extension version */
+static char *citusAvailableVersion = NULL;
```

Comment:
`availableMajorVersion` is all we need for this variable (no need for full extension version, we always split on `-` anyhow).